Updating SAT version

shivaprasad-metimath opened the pull request:
IM:CRAYSAT-1795
Reviewer:Ryan

## Summary and Scope

Security vulnerabilities have been addressed in the latest version of SAT. Hence bumping the version


## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [CRAYSAT-1795](https://jira-pro.it.hpe.com:8443/browse/CRAYSAT-1795)

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * MUG

### Test description:

The SAT commands are validated and it looks good without any issues


## Risks and Mitigations

Low risk


## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

Commit:
Updating SAT version

IM:CRAYSAT-1795
Reviewer:Ryan

haasken-hpe reviewed (approved):
Looks good, but if I have time before this is merged, I'm going to update it to 3.25.9 as I have backported the more important part of the fix for CRAYSAT-1800 to the `release/3.25` branch of SAT, and I'd like to get that fix included in CSM 1.5 as well.

haasken-hpe committed:
CRAYSAT-1800: Update cray-sat to 3.25.9

Update the version of the cray-sat container image to 3.25.9 to include
the fix for the traceback in CRAYSAT-1800.

Test Description:
This version of the container image was tested on mug. The `sat showrev`
command was tested both with a valid and invalid filter, and `sat
bootprep run` was tested against an input file with multiple errors to
confirm that each line of a multi-line error message is prefixed with
the log level for compatibility with the IUF.
